README.md

```diff
@@ -47,18 +47,18 @@ Please see full version in [example](./example).
 Exposed command in a nutshell:
 
 ```javascript
-cySubject.upload(file, options);
+cySubject.upload(fileOrArrayOfFiles, processingOptions);
 ```
 
-`file` is an object that represents file information and contains following properties:
+`fileOrArrayOfFiles` is an object (or an array of those) that represents file information and contains following properties:
 
-- (string) `fileContent` – raw file content
-- (string) `fileName` – file name with extension
-- (string) `mimeType` – file mime type (string)
+- {String} `fileContent` – raw file content
```

Comment:
Why did `String` become capitalized?

Reply:
It looks more like jsdoc now.
As for me it's a bit more fancy to have it capitalized, and officially it does not matter:
https://github.com/jsdoc3/jsdoc/issues/1046